org.naev.Naev.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "org.naev.Naev",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "naev",
+  "rename-icon": "naev",
+  "rename-desktop-file": "naev.desktop",
+  "rename-appdata-file": "naev.appdata.xml",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--device=dri"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    {
+      "name": "libzip",
+      "buildsystem": "cmake-ninja",
+      "cleanup": [ "/bin", "/share" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://libzip.org/download/libzip-1.5.1.tar.xz",
+          "sha256": "04ea35b6956c7b3453f1ed3f3fe40e3ddae1f43931089124579e8384e79ed372"
+        }
+      ]
+    },
+    {
+      "name": "libluajit",
+      "no-autogen": true,
+      "make-args": ["PREFIX=/app"],
+      "make-install-args": ["PREFIX=/app"],
+      "cleanup": ["/bin"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://luajit.org/download/LuaJIT-2.0.5.tar.gz",
+          "sha256": "874b1f8297c697821f561f9b73b57ffd419ed8f4278c82e05b48806d30c1e979"
+        }
+      ]
+    },
+    {
+      "name": "naev",
+      "config-opts": [
+        "--disable-debug",
```

Comment:
Debug symbols should stay enabled.

Reply:
Why? A debug build would be slower than a release build.